packages/apps/human-app/server/src/integrations/exchange-oracle/spec/exchange-oracle.gateway.spec.ts

```diff
@@ -92,7 +92,9 @@ describe('ExchangeOracleApiGateway', () => {
   describe('fetchOracleStatistics', () => {
     it('should successfully call the requested url for oracle statistics', async () => {
       const details = oracleStatsDetailsFixture;
-      nock(statisticsExchangeOracleUrl).get('/stats').reply(200);
+      nock(statisticsExchangeOracleUrl).get('/stats')
+        .matchHeader('Authorization', `Bearer ${details.token}`)
```

Comment:
this test doesn't check implementation: 
in your test, you add the 'Bearer ' part to the details.token - this not match actual implementation

Reply:
Issue: 1ed59b1b Reverted ("Add token authentication for Oracle statistics.")